scripts/webpack.config.build.js

```diff
@@ -2,16 +2,16 @@ const path = require('path');
 const { CleanWebpackPlugin } = require('clean-webpack-plugin');
 // const nodeExternals = require('webpack-node-externals');
 
-const { NODE_ENV, SERVICE, AWS_PROVIDER_REGION, AWS_PROVIDER_STAGE } = process.env;
+const { NODE_ENV, SERVICE, AWS_REGION, AWS_PROVIDER_STAGE } = process.env;
```

Comment:
delete this file?

Reply:
Very likely going to be deleted. There is a ticket next sprint to clean up redundant packages and config.